packages/html/src/switch/switch.spec.tsx

```diff
@@ -11,6 +11,7 @@ const states = [
 
 const options = {
     size: [ Size.small, Size.medium, Size.large ],
+    roundness: [ null, "sm", "md", "lg", "full" ]
```

Comment:
use from misc - [see Button for ref](https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/button/button.spec.tsx#L16)